.github/workflows/ticdc_canal_json_extension.yaml

```diff
@@ -0,0 +1,83 @@
+name: CDC Canal-JSON-Extension Tests
```

Comment:
Can we integrate it into the original test, we no longer have the resources to support the new actions.